requirements.txt

```diff
@@ -6,3 +6,9 @@ ipykernel~=6.16
 
 # project packages
 pandas~=1.4
+plotly~=5.17.0
+pandas~=2.0.3
+plotly~=5.18.0
```

Comment:
should never have the same package twice in a requirements.txt. This docker container will never build. You should be checking that the container builds